x/multi-staking/keeper/abci.go

```diff
@@ -15,7 +16,8 @@ func (k Keeper) BeginBlocker(ctx sdk.Context) {
 }
 
 // need a way to better name this func
-func GetUnbondingHeightsAndUnbondedAmounts(ctx sdk.Context, unbondingDelegation stakingtypes.UnbondingDelegation) map[int64]math.Int {
```

Comment:
https://github.com/cosmos/cosmos-sdk/pull/16324) `NewKeeper` now takes a `KVStoreService` instead of a `StoreKey`, and methods in the `Keeper` now take a `context.Context` instead of a `sdk.Context`